message-index/messages/GHC-11861/example2/index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Empty character literals
+summary: Incorrect TemplateHaskell syntax
+introduced: 9.6.1
+severity: error
+---
+
+Empty character was given in a single quote. This is not a valid literal.
+
+If you use TemplateHaskell `''T` syntax is valid and:
+> ''T has type Name, and names the type constructor T. That is, ''⟨thing⟩ interprets ⟨thing⟩ in a type context.
```

Comment:
What is this quoting? I think that block quotations should have a link to a source, but I'm not sure this needs to be a quotation.

Reply:
What about the phrasing:

When Template Haskell is enabled, `''T` is syntax that refers to a quoted name of the type `T`. In this case, `''` is not an empty character literal, but rather part of the quotation syntax for type names.

Reply:
Rephrased and added link to template Haskell documentation part for TemplateHaskellQuotes.